src/vss_tools/vspec/vssexporters/vss2samm/helpers/namespaces.py

```diff
@@ -21,22 +20,20 @@ def get_node_name_from_vspec_uri(node_uri: URIRef):
 
 
 def get_unit_uri(unit_name: str):
-    return URIRef(f"{samm_base_namespace}:unit:{cfg.SAMM_VERSION}#{unit_name}")
+    return URIRef(f"{samm_base_namespace}:unit:{Config.SAMM_VERSION}#{unit_name}")
 
 
-log.debug("VSS to SAMM CONFIG:\n  -- SAMM_TYPE   : %s\n  -- SAMM_VERSION: %s\n", cfg.SAMM_TYPE, cfg.SAMM_VERSION)
-
 # NOTE: samm_base_namespace is more for the ESMF core libraries
 samm_prefix = "urn:samm"
 samm_base_namespace = f"{samm_prefix}:org.eclipse.esmf.samm"
 Namespaces = {
-    "samm": f"{samm_base_namespace}:meta-model:{cfg.SAMM_VERSION}#",
-    "samm-c": f"{samm_base_namespace}:characteristic:{cfg.SAMM_VERSION}#",
-    "samm-e": f"{samm_base_namespace}:entity:{cfg.SAMM_VERSION}#",
-    "unit": f"{samm_base_namespace}:unit:{cfg.SAMM_VERSION}#",
+    "samm": f"{samm_base_namespace}:meta-model:{Config.SAMM_VERSION}#",
+    "samm-c": f"{samm_base_namespace}:characteristic:{Config.SAMM_VERSION}#",
+    "samm-e": f"{samm_base_namespace}:entity:{Config.SAMM_VERSION}#",
+    "unit": f"{samm_base_namespace}:unit:{Config.SAMM_VERSION}#",
 }
 
 # Below formatted namespace should look like: urn:samm:com.covesa.vss.spec:5.0.0#
```

Comment:
As per above, when this script is loaded, before the Config is initialized in the vss2samm, this will lead to `@prefix : <urn:samm:None:None#> .`  in the generated aspect models, which will raise a SAMM error. 

Expected result should be like: `@prefix : <urn:samm:com.covesa.vss.spec:5..0.0#> .`

The config, and more specifically: OUTPUT_NAMESPACE, VSPEC_VERSION and SPLIT_DEPTH fields must not be Null before config is loaded by any other script, i.e. we should **init** the config, and then dynamically load other helpers, which then will load concepts and namespaces, where the above config fields are used.